File: 240322/괄호 쌍 만들어주기 2/pair-parentheses-2.py
line = input().strip()

count = 0
for i in range(len(line)-1):
    if line[i] == '(' and line[i+1] == '(':
        for j in range(i+2,len(line)-1):
            if line[j] == ')' and line[j+1] == ')':
                count += 1

print(count)

File: 240404/괄호 쌍 만들어주기 2/pair-parentheses-2.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
    char line[101];
    int length, count = 0;

    scanf("%s",line);
    length = strlen(line);

    for(int l = 0; l < length-1; l++)
        if (line[l] == '(' && line[l+1] == '(')
            for(int r = l + 2; r < length-1; r++)
                if (line[r] == ')' && line[r+1] == ')')
                    count++;

    printf("%d\n",count);
    return 0;
}